lib/rules/no-extend-native.js

```diff
@@ -21,6 +21,13 @@ var BUILTINS = [
 
 module.exports = function(context) {
 
+    var config = context.options[0] || {};
+    var exceptions = config.exceptions || [];
+
+    var modifiedBuiltins = BUILTINS.filter(function(builtIn) {
+        return exceptions.indexOf(builtIn) === -1 ? true : false;
```

Comment:
You don't need `? true : false ` on this line. The expression already evaluate to true or false.